src/Shared/Recruit.Vacancies.Client/Application/Queues/Messages/CreateEmployerDashboardQueueMessage.cs

```diff
@@ -0,0 +1,7 @@
+namespace Esfa.Recruit.Vacancies.Client.Application.Queues.Messages
+{
+    public class CreateEmployerDashboardQueueMessage
+    {
```

Comment:
Could we not have a general class for EmployerAccountId or VacancyReference queue messages rather than specific to the job as the queues are separate and provide that separation.

Reply:
The generic query store uses the message type to determine which queue to add the message to so each queue needs its own message class.

Reply:
👍